libs/utils/languages.js

```diff
@@ -0,0 +1,22 @@
+export default {
+  en: {
+    ietf: 'en',
+    tk: 'hah7vzn.css',
+    regions: [
+      { region: 'us', ietf: 'en-US', tk: 'hah7vzn.css' },
```

Comment:
would it be possible to rely on the lang based typekit by default and only set a custom one if needed? This way we would not need to set the same typekit for every region.

Reply:
definitely!